views/scss/admin.scss

```diff
@@ -2,7 +2,45 @@
 @import "inc/buttons";
 
 .permission-container {
-    table td {
-        min-width: 130px;
+    .grid-container {
+        margin-top: 40px;
+        margin-bottom: 20px;
+    }
+
+    .msg-edit-area {
+        border: 1px $uiGeneralContentBorder solid;
+        background: $canvas;
+        padding: 20px 20px 10px 20px;
+        @include simple-border();
+
+        .icon-add {
+            font-size: 2rem !important;
```

Comment:
There's a `font-size` (check for the exact name) mixin that you should use instead